ouroboros-network/src/Ouroboros/Network/BlockFetch/Decision.hs

```diff
@@ -499,12 +507,12 @@ filterNotAlreadyInFlightWithOtherPeers FetchModeBulkSync chains =
 
 prioritisePeerChains
   :: HasHeader header
-  => (ChainFragment header -> ChainFragment header -> Ordering)
+  => (AnchoredFragment header -> AnchoredFragment header -> Ordering)
   -> (header -> SizeInBytes)
-  -> [(FetchDecision [ChainFragment header], PeerFetchInFlight header,
-                                             PeerGSV,
-                                             peer)]
-  -> [(FetchDecision [ChainFragment header], peer)]
+  -> [(FetchDecision (CandidateFragments header), PeerFetchInFlight header,
+                                                  PeerGSV,
+                                                  peer)]
+  -> [(FetchDecision (CandidateFragments header), peer)]
```

Comment:
Yep, good, so now we at least have the right info available to prioritise, not just the filtered fragments.